Source/CBL_Replicator.m

```diff
@@ -341,13 +341,6 @@ - (void) start {
                  }
                 ];
 
-    // If client didn't set an authorizer, use basic auth if credential is available:
-    if (!_authorizer) {
-        _authorizer = [[CBLBasicAuthorizer alloc] initWithURL: _remote];
```

Comment:
Same here — pulling credential out of the URL string should work with both basic and digest auth.